### DISCUSSION TITLE
Ability to sort on multiple fields in REST API

### DISCUSSION THREAD

**Discussion by hades200082** [payloadcms/payload#2089] [Feature Requests & Ideas]
Ability to sort by one field, then another, etc. when using the REST API. 

In SQL terms I'm looking for the equivalent of `SORT BY FieldA DESC, FieldB ASC`

**Comment by jobsteven** [comment]
surely, this is a very common feature

**Reply by hades200082** [reply]
Apparently only you and I think it's required :(

**Reply by r1tsuu** [reply]
i also very need this and have added a PR #3236

**Comment by edwardbc** [comment]
+1 ! I'm also having this need (which is also duplicated on this other discussion
https://github.com/payloadcms/payload/discussions/2409
)

**Comment by DanRibbens** [comment]
Merged 🎉  it will be in the next beta release.
A huge thank you to @andershermansen for picking this up and running with it!
